RFC-004-revising-licenses-page.md

```diff
@@ -0,0 +1,118 @@
+# RFC004 - Revising website licenses terms/page
+
+## Proposed by
+
+Ryan Macklin ([@macklin](https://thegooddocs.slack.com/team/U01DYRWG43X))
+Initially submitted on 2021-07-13
+
+## Current status
+
+- [ ] Draft
+- [x] Under discussion (until 2021-07-27, assuming two weeks post initial submission)
+- [ ] Final comment and voting (until YYYY-MM-DD) {{Add date after selecting this status.}}
+- [ ] Accepted
+- [ ] Rejected
+- [ ] Implemented
+- [ ] Deferred
+- [ ] Withdrawn
+
+## Proposal overview
+
+I believe we should rethink the licensing terms for website content, as listed on [thegooddocsproject.dev/licenses/](https://thegooddocsproject.dev/licenses/). As they're currently written, they provision everything as [Creative Commons Attribution 4.0](https://creativecommons.org/licenses/by/4.0/) (a.ka. "CC BY 4.0), including all text, images, HTML code, and anything else.
```

Comment:
I think it would help to list specifically want you want to protect. You've mentioned personal information in the past, such as personal photos. (I'd be open to excluding personal photos, which we could do in the back end by putting photos into a separately licensed repository, or maybe in a separately licensed directory.)
Is there any other personal information which is printed in blogs which we need to protect?
If the author is sharing blog material themselves, they are in control about what is shared under the CC-By license.

Reply:
Acknowledged. I'll step back from my comment.